.bazelrc

```diff
@@ -25,3 +25,12 @@ build --build_metadata=TEST_GROUPS=//packages
 ## Reset remote cache and backend support
 build:offline --bes_backend="" --remote_cache=""
 run:offline --bes_backend="" --remote_cache=""
+
+# Load any settings specific to the current user.
+# .bazelrc.user should appear in .gitignore so that settings are not shared with team members
+# This needs to be last statement in this
+# config, as the user configuration should be able to overwrite flags from this file.
+# See https://docs.bazel.build/versions/master/best-practices.html#bazelrc
+# (Note that we use .bazelrc.user so the file appears next to .bazelrc in directory listing,
+# rather than user.bazelrc as suggested in the Bazel docs)
+try-import %workspace%/.bazelrc.user
```

Comment:
@mistic, want to confirm you're OK with moving this here - otherwise we are unable to override the token on CI.

The order will be:
* common
* bazelrc
* bazelrc.user

Reply:
I'm not getting the reasoning why `.bazelrc.user` will affect the CI override. CI configs are located at https://github.com/elastic/kibana/blob/master/src/dev/ci_setup/.bazelrc-ci and we don't need to touch it with that change. Do you mean overriding the token locally with an user configuration?

Reply:
Oh, I see we are completely overwriting the `.bazelrc`. I thought we were `.bazelrc.user` on CI. I will need to look into it further on Monday to understand why this was happening then.